resources_pages/lab_submission.md

```diff
@@ -7,46 +7,36 @@ subtitle: And lab submission guidelines
 ### Student repositories
 For every lab, we will create a repository (repo) for each student. For example,
 if your CWL is `goatcabin` then for DSCI 521 lab 1 there will be a repository called
-`DSCI_521_lab1_goatcabin`. At the start of the program each of you will be given
-a "home" repository in GitHub, which will help you find the various repositories.
+`DSCI_521_lab1_goatcabin`. To see a list of all repositories that you have access to,
+navigate to the homepage of your year's organization; for example, for the 2020-21 cohort,
+go to <https://github.ubc.ca/mds-2020-21>.
 
 NOTE: please do not confuse your personal lab-specific repository with the general course
 repository, which will be named something like `DSCI_521_platforms-dsci_students`. That
 repo is public to all students and is where you can access lectures, due dates, readings, etc.
 
 ### Submission
 We anticipate that you will clone your lab-specific repository and do your work from
-within there. When you push something back to <https://github.ubc.ca/> then you have submitted the assignment.
+within there. To submit your labs you must **both** push your work to <https://github.ubc.ca/> 
+**and** submit your rendered HTML file to Canvas. Your assignment will come bundled with a short script
+to render to HTML and push your work to Canvas. 
 You can push changes as many times as you want before the deadline; only the final version will be graded.
 
 ### Deadline
 The lab deadlines are given on the course repo and/or MDS calendar. **These are hard deadlines.** In particular,
-when the deadline passes you will lose write access to the repository.
+when the deadline passes you will lose write access to the repository. The default deadline for labs is
```

Comment:
So can they still submit a lab late? Or is that only on a case to case basis where they email the instructor?